public/style/morestyles.css

```diff
@@ -0,0 +1,9 @@
+.center {
```

Comment:
Is this file produced by scss and node-sass? If it is, I think it should be ignored.

Reply:
No, this is an extra file that I am using to write custom css classes that I had been re-using quite a bit. If I make a small modification that I'll never use again, then I've been doing that in an in-line style tag on the component. If I find that I am making the same in-line style change in several different areas, then I make that into a class and write it here. So far, styling has been a combo of pure bulma, in-line css mods, custom classes like in here, and bulma.sass mods (mostly to change default colors, fonts).

Reply:
The file produced by node-sass is style.css, the large one that probably won't load

Reply:
Got it! Thanks!